lib/ui/text.dart

```diff
@@ -274,6 +281,7 @@ class TextStyle {
   /// * `fontSize`: The size of glyphs (in logical pixels) to use when painting the text.
   /// * `letterSpacing`: The amount of space (in logical pixels) to add between each letter.
   /// * `wordSpacing`: The amount of space (in logical pixels) to add at each sequence of white-space (i.e. between each word).
+  /// * `textBaseline`: The primary baseline along which this text should be aligned.
```

Comment:
The common baseline that should be aligned between this text span and its parent text span, or, for the root text spans, with the line box.